site-content/source/modules/ROOT/pages/blog.adoc

```diff
@@ -16,7 +16,7 @@ NOTES FOR CONTENT CREATORS
 [discrete]
 === Join Cassandra at Apachecon 2021
 [discrete]
-==== August 27, 2021
+==== September 20, 2021
```

Comment:
This is already fixed on the site by [CASSANDRA-16980](https://issues.apache.org/jira/browse/CASSANDRA-16980) (PR #73).

Reply:
This version of `blog.adoc` is out of date and needs to be re-based from latest version in `trunk`.